internal/buildscripts/packaging/installer/install.sh

```diff
@@ -390,12 +403,49 @@ configure_fluentd() {
   fi
 }
 
+update_instrumentation_config() {
+  local deployment_environment="$1"
+
+  if [ -f "$instrumentation_config_path" ]; then
+    echo "Adding 'deployment.environment=${deployment_environment}' resource attribute to $instrumentation_config_path"
+    if grep -q '^resource_attributes=' "$instrumentation_config_path"; then
```

Comment:
just a thought
```
sed -e "s|\(resource_attributes=.*deployment.environment=\)\([^,]*\)\(.*\)|\1$deployment_environment\3|g"
```

Reply:
Nice!  I originally thought about using something like that, but would have needed to use multiple complex sed/grep expressions to account for all of the other conditions.  So just thought an iterative method would be more straightforward.